source/client/controllers/login-controller.js

```diff
@@ -8,7 +8,8 @@ Space.messaging.Controller.extend(Space.accountsUi, 'LoginController', {
   eventSubscriptions() {
     return [{
       'Space.accountsUi.LoginRequested': this._onLoginRequested,
-      'Space.accountsUi.LogoutRequested': this._onLogoutRequested
+      'Space.accountsUi.LogoutRequested': this._onLogoutRequested,
+      'Space.accountsUi.LoginWithGoogleRequested': this._onLoginWithGoogleRequested,
```

Comment:
There are two concerns with that. Login and account creation are one if the user is logging in with external OAUTH service, and events have different properties.